apps/SageAccountWeb/src/AppInitializer.tsx

```diff
@@ -50,9 +52,9 @@ function AppInitializer(props: { children?: React.ReactNode }) {
   useEffect(() => {
     if (sourceApp?.palette) {
       setTheme(
-        createTheme(
-          deepmerge(defaultThemeOptions, { palette: sourceApp.palette }),
-        ),
+        createTheme(defaultMuiThemeOptions, sageAccountWebThemeOverrides, {
+          palette: sourceApp.palette,
+        }),
```

Comment:
Merging and creating the themes in one place IMO makes it more clear how the theme overrides are applied.